terms.html

```diff
@@ -53,6 +59,8 @@
   <dd>
 A thing with distinct and independent existence, such as a person,
 organization, or device that performs one or more roles in the ecosystem.
+For accountability and other purposes, it can be relevant to know 
+whether an entity performs such roles by itself, or on behalf of some other entity.
```

Comment:
why does this matter for "entity" are we saying that "issuers" have "issuers"? 

I am confused by how this helps implementers understand the data model.

Reply:
No, we're not saying that "issuers" have "issuers". What it tries to say that if you have an issuer as currently defined, this could be a person, organization or a device (per the definition). For VCs issued by individuals or organizations, people would easily infer that the individual or organization could be held accountable. If a VC is issued by a device (e.g. a camera, or sensor in the field), accountability can not be inferred by means defined in this document.

Reply:
What's confusing you? 

The text clearly states that the term 'entity' is basically anyone or anything that can perform the role of issuer, holder or verifier, and it states that it can be a person, an organization, or a device. To me, that's quite clear.

I can see though that this meaning deviates from 'normal' American language, e.g., as documented by [Merriam-Webster](https://www.merriam-webster.com/dictionary/entity). That should, however, not confuse us (if that is what is going on). We _define_ the terms that we need in this standard, because we need to make sure that anyone can use the terms in the meanings in which they are defined, and diligent readers can interpret them in that meaning, and not be confused because of other stuff that influences their interpretations.

This is a very common practice: It's what standardization orgs do (ISO has terminology standards, e.g., ISO 27000 for information security). It's also what regulators do: they specify the terminology for a regulation in one of its first clauses so lawyers and judges can agree on whether or not someone or something is, e.g., a 'driver'. It's what software engineers do (you include header files in C, or you import namespaces in other languages). It's what we need, too.

Reply:
```suggestion
whether an entity (for instance, a person) performs such roles by itself, 
or on behalf of some other entity (for instance, an organization).
```